src/_playground/Routes.js

```diff
@@ -387,9 +387,9 @@ export class Routes extends Component {
                                     inputValue={this.state.query}
                                     onChange={this.onChangeHandler} />
                             </div>
-                            <nav className='frDocs-Nav'>
+                            <SideNav selectedId='Home'>
```

Comment:
I don't know if this should be hard-coded.  It seems as though there should be a different state variable to track this.  The state variable starts as `'Home'`, but should change each time a new item is selected.

That said, just looking at the `SideNav` component, it looks like there is no way to get that information out of the component to track it as a consumer.  The component takes `selectedId` as a prop and tracks its own state, but it would be very easy for `SideNav` and its consumer to get out of sync.  I think there should be an `onItemSelect` callback function prop on the `SideNav` component that a consumer could subscribe to.

I guess this was one of our goals -- eating our own dog food. 🐶 

We should create a new story to fix `SideNav` and then once that is complete, we can make additional adjustments to this story.